runners/katacoda/templates/cloneRepository.md

```diff
@@ -0,0 +1,17 @@
+<%= text; %>
+
+<%= cdCommand; %>
+
+<% if (string) { %>
+If the parent directories aren't already in the project, 'mkdir -p' will create them for you. 
```

Comment:
Can't we simply check if directoryPath is not empty? Then we don't need this extra variable.

Reply:
Can't we just use 'git clone <repository> <path>' to create the directories? I believe that it will also generate the missing directories.

Reply:
git clone [repo] [path]

Reply:
Yes, that will work too. Maybe it's better this way. Then the user has to change the directory only once.